### PR TITLE
Fix missing arg added debug arg

### PR DESCRIPTION
**Pull request by escamoteur**
## 1.0.6

 - add `--debug` flag to debug signing issues, fixed missing argument `/f' for not '.pfx' certificates

### CODE REVIEW

**Comment by escamoteur** [comment]
Thanks for this awesome package!

**Review by YehudaKremer — COMMENTED** [review]
more debug information, very nice addition:
image

File: lib/src/configuration.dart [hunk 1]
```diff
@@ -72,7 +76,8 @@ class Configuration {
     var parser = ArgParser()
       ..addOption('password', abbr: 'p')
       ..addOption('certificate', abbr: 'c')
-      ..addOption('version', abbr: 'v');
+      ..addOption('version', abbr: 'v')
+      ..addFlag('debug');
```
Comment (YehudaKremer) [inline lib/src/configuration.dart:80]:
```suggestion
      ..addFlag('debug', abbr: 'd');
```
should we add short flag?

**Review by YehudaKremer — APPROVED** [review]
@escamoteur thank you for this contribution

**Comment by YehudaKremer** [comment]
Version 1.0.6 in now available on pub.dev

**Comment by escamoteur** [comment]
your code was really easy to follow :-)

**Pull request merged**